docs/pages/setup/admin/trustedclusters.mdx

```diff
@@ -71,51 +72,96 @@ $ tsh ssh --cluster=leaf host
   type="tip"
   title="Load Balancers"
 >
-  The scheme above also works even if the "root" cluster uses multiple proxies behind a load balancer (LB) or a DNS entry with multiple values.
-  This works by "leaf" establishing a tunnel to *every* proxy in "root". This requires that an LB uses a round-robin or a similar balancing algorithm. Do not use sticky load balancing algorithms (a.k.a. "session affinity" or "sticky sessions") with
-  Teleport proxies.
+
+  The scheme above also works even if the "root" cluster uses multiple proxies
+  behind a load balancer (LB) or a DNS entry with multiple values. This works by
+  "leaf" establishing a tunnel to *every* proxy in "root".
+  
+  This requires that an LB use a round-robin or a similar balancing algorithm.
+  Do not use sticky load balancing algorithms (a.k.a. "session affinity" or
+  "sticky sessions") with Teleport Proxies.
+
 </Admonition>
 
 ## Join Tokens
 
-Lets start with the diagram of how connection between two clusters is established:
+Let's start with a diagram of how a connection between two clusters is established:
 
 ![Tunnels](../../../img/trusted-clusters/TrustedClusters-Simple.svg)
 
-The first step in establishing a secure tunnel between two clusters is for the *leaf* cluster "leaf" to connect to the *root* cluster "root". When this
-happens for *the first time*, clusters know nothing about each other, thus a shared secret needs to exist for "root" to accept the connection from "leaf".
+The first step in establishing a secure tunnel between two clusters is for the
+*leaf* cluster "leaf" to connect to the *root* cluster "root". When this happens
+for *the first time*, clusters know nothing about each other, thus a shared
+secret needs to exist for "root" to accept the connection from "leaf".
+
+This shared secret is called a "join token". 
+
+Before following these instructions, you should make sure that you can connect
+to Teleport.
 
-This shared secret is called a "join token". There are two ways to create join tokens: to statically define them in a configuration file or to create them on the fly using `tctl` tool.
+(!docs/pages/includes/tctl.mdx!)
+
+<Tabs>
+<TabItem scope={["oss", "enterprise"]} label="Self-Hosted">
+
+There are two ways to create join tokens: to statically define them in a
+configuration file or to create them on the fly using the `tctl` tool.
 
 <Admonition
   type="tip"
   title="Important"
 >
-  It's important to note that join tokens are only used to establish the connection for the first time. The clusters will exchange certificates and won't use tokens to re-establish their connection afterward.
+
+  It's important to note that join tokens are only used to establish a
+  connection for the first time. Clusters will exchange certificates and
+  won't use tokens to re-establish their connection afterward.
+
 </Admonition>
 
-### Static Join Tokens
+### Static join tokens
 
 To create a static join token, update the configuration file on "root" cluster
```

Comment:
Should say on **the** "root" cluster.

Reply:
Although I'm not sure static join token section is applicable to cloud (yet), as customers cannot edit cloud's teleport.yaml , right?

Reply:
I've aded a `Tabs` component with the `scope` property set around this H3 and the "Dynamic Join Tokens" H3, so these sections only appear if the reader has selected "OpenSource" or "Enterprise" in the dropdown menu.